src/TTrigger.cpp

```diff
@@ -194,28 +199,32 @@ bool TTrigger::setRegexCodeList(QStringList regexList, QList<int> propertyList)
         if (propertyList.at(i) == REGEX_PERL) {
             const char* error;
             const QByteArray& regexp = regexList.at(i).toUtf8();
+            const QString& regexp16 = regexList.at(i);
 
             int erroffset;
 
             // PCRE_UTF8 needed to run compile in UTF-8 mode
             // PCRE_UCP needed for \d, \w etc. to use Unicode properties:
-            QSharedPointer<pcre> re(pcre_compile(regexp.constData(), PCRE_UTF8 | PCRE_UCP, &error, &erroffset, nullptr), pcre_deleter);
+            QSharedPointer<pcre> re(pcre_compile(regexp.data(), PCRE_UTF8 | PCRE_UCP, &error, &erroffset, nullptr), pcre_deleter);
```

Comment:
`pcre_compile(...)` doesn't change the first argument does it? So is the change from `.constData()` to `.data()` intentional here?

Reply:
No, it's unintentional. I can't use git and this is unwanted consequence.